<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFWControl</class>
 <widget class="QDialog" name="CFWControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Wheel Control</string>
  </property>
  <widget class="QProgressBar" name="progressBar_cfwRunning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cfwPos">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setupCFW">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Position">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>93</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_curFilterPos">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Moving ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_curFilterName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>93</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_setCFWPos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Position:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
